test/framework/containers.py

```diff
@@ -492,7 +505,7 @@ def test_container_config_template_recipe(self):
             "from: foobar",
             "",
             "%post",
-            "eb toy-0.0.eb --robot --installpath=/app/ --prefix=/scratch --tmpdir=/scratch/tmp",
+            "eb toy-0.0.eb --robot --debug -l",
```

Comment:
what happened to those dropped arguments?

Reply:
I'm just changing the template recipe being used here, to make it a more realistic use case we're testing with. The test would still work without this change.

Configuration options like `--installpath` are now handled through the corresponding environment variable (cfr. the changes in #2913), which allows for overriding those settings by injecting another value via the corresponding CLI option through `eb_args`.